src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -1105,6 +1106,9 @@ public override object VisitField(FieldSymbol symbol, TypeCompilationState argum
                                 chain,
                                 previousSubmissionFields,
                                 compilationState,
+                                false,
```

Comment:
It's not clear what this means. Could you use named arguments or define a local constant?